three.js/src/threex/threex-artoolkitcontext.js

```diff
@@ -255,7 +255,7 @@ ARjs.Context.prototype.getProjectionMatrix = function (srcElement) {
     var projectionMatrix = new THREE.Matrix4().fromArray(projectionMatrixArr)
 
     // apply context._axisTransformMatrix - change artoolkit axis to match usual webgl one
-    projectionMatrix.multiply(this._artoolkitProjectionAxisTransformMatrix)
+    //projectionMatrix.multiply(this._artoolkitProjectionAxisTransformMatrix)
```

Comment:
We need to remove this line because projection matrix from camera is giving different values in the new artoolkit.min.js unlike the old version: